### DISCUSSION TITLE
Add a way to duplicate Notes

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#840] [Feature Requests]
There should be some easy way to duplicate a note.

---
> Please vote on this issue with a 👍 reaction. It's one of the primary mechanisms used to prioritize issues. Please also subscribe to the issue if you're interested, as that way I can discuss possible implementations and share mockups.

**Comment by aisbergde** [comment]
or give access to the file system I we can use any file manager.
But of course it would be better, if duplicate is available directly in the app.
